tools/update-goldens

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Nice! should you refer to this in schema-generator-test instead of what we have there today? i.e.
```
Run the following command to update the golden file:

    cp "bazel-bin/${data.generated}" "${data.golden}"
```

Reply:
Let me disambiguate: You're saying that I should update the error message when the golden tests fail? If so, I updated that (the diff is listed above). :)